src/lib.rs

```diff
@@ -1,5 +1,25 @@
 #![no_std]
 #![deny(missing_debug_implementations)]
+#![deny(missing_docs)]
+
+//! An experimental Multiboot 2 crate for ELF-64/32 kernels.
+//!
+//! The GNU Multiboot specification aims provide a standardised
```

Comment:
Should this read "The GNU Multiboot specification aims **to** provide a standardised"?